src/main/java/ValidationResults.java

```diff
@@ -0,0 +1,39 @@
+import lombok.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+
+public class ValidationResults {
+
+    public int total;
```

Comment:
Having public fields in your class breaks encapsulation, so all of them should be  with private  modifier and accessed via getters/setters